thepeg.sh

```diff
@@ -6,29 +6,33 @@ requires:
   - Rivet
   - GSL
   - fastjet
+  - pythia8
+  - HepMC
+  - lhapdf5
+  - cgal
+  - boost
 ---
 #!/bin/bash -e
 
-Pythia8='/cvmfs/alice.cern.ch/x86_64-2.6-gnu-4.1.2/Packages/pythia/v8186'
-HepMC='/cvmfs/alice.cern.ch/x86_64-2.6-gnu-4.1.2/Packages/HepMC/v2.06.09'
-LhaPDF='/cvmfs/alice.cern.ch/x86_64-2.6-gnu-4.1.2/Packages/lhapdf/v5.9.1'
-Cgal='/cvmfs/alice.cern.ch/x86_64-2.6-gnu-4.1.2/Packages/cgal/v4.4'
-Boost='/cvmfs/alice.cern.ch/x86_64-2.6-gnu-4.1.2/Packages/boost/v1_53_0'
+export LD_LIBRARY_PATH="${LHAPDF5_ROOT}/lib:${CGAL_ROOT}/lib:${BOOST_ROOT}/lib:${LD_LIBRARY_PATH}"
+export LIBRARY_PATH="${LHAPDF5_ROOT}/lib:${CGAL_ROOT}/lib:${BOOST_ROOT}/lib:${LIBRARY_PATH}"
+export LHAPATH="${LHAPDF5_ROOT}/share/lhapdf"
+export PERLLIB=/usr/share/perl5
```

Comment:
The AliEn-Runtime package contains a version of perl5, but you can leave it like this, I'll take care of it.